Vizzuality/trase
# Update version modal

## Asana

https://app.asana.com/0/0/1200047516960739/f
https://app.asana.com/0/1198886845396491/1199905805725198/f
## Description

![image](https://user-images.githubusercontent.com/9701591/110785178-ec727380-826a-11eb-9d2d-0afd2180137e.png)

Simplify current modal
Add context 0 information for temporary general information
Update styles

## Testing instructions

Be sure that you have ENABLE_VERSIONING turned on and go to any tool context. You should see the methods and data option
